frontend/integration-tests/views/login.view.ts

```diff
@@ -9,15 +8,11 @@ export const userDropdown = $('[data-test=user-dropdown] .pf-c-app-launcher__tog
 
 export const selectProvider = async(provider: string) => {
   const idpLink = element(by.cssContainingText('.idp', provider));
-  while (!(await idpLink.isPresent())) {
-    await browser.get(appHost);
-    await browser.sleep(3000);
-  }
   await idpLink.click();
 };
 
 export const login = async(providerName: string, username: string, password: string) => {
-  if (providerName) {
+  if (await $('a.idp').isPresent()) {
```

Comment:
okay.

Reply:
We should not be creating a separate kubevirt login imo unless it's absolutely necessary.

Reply:
Some of kubevirt tests are invoking `execSync` to do some operations from cli to save time, to achieve this it need to do `oc login` within the login scenario.

@rhrazdil I think maybe we should use `yaml` dialog or VM wizard to create all resources instead of doing it from command line? So we can share one login scenario.

Reply:
I revert this change anyway.